pinot-core/src/test/java/org/apache/pinot/core/operator/transform/function/CaseTransformFunctionTest.java

```diff
@@ -106,7 +106,8 @@ public void testCaseTransformFunctionWithIntResults() {
       testCaseQueries(String.format("%s(%s, %s)", functionType.getName(), LONG_SV_COLUMN,
           String.format("%d", _longSVValues[INDEX_TO_COMPARE])), getPredicateResults(LONG_SV_COLUMN, functionType));
       testCaseQueries(String.format("%s(%s, %s)", functionType.getName(), FLOAT_SV_COLUMN,
-          String.format("%f", _floatSVValues[INDEX_TO_COMPARE])), getPredicateResults(FLOAT_SV_COLUMN, functionType));
+              "CAST (" + String.format("%f", _floatSVValues[INDEX_TO_COMPARE]) + " AS FLOAT)"),
```

Comment:
(nit) Usually there is no space after `CAST`
```suggestion
              "CAST(" + String.format("%f", _floatSVValues[INDEX_TO_COMPARE]) + " AS FLOAT)"),
```

Reply:
resolved